content/en/docs/storage-providers/seal-workers.md

```diff
@@ -204,73 +204,100 @@ Worker 1, host othercomputer
+- Worker 1 (PC1, PC2):
+```json
+{
+  "ID": "2546c5be-10f0-aa96-906b-24434a6c94a0",
+  "Weight": 10,
+  "CanSeal": true,
+  "CanStore": false,
+  "MaxStorage": 0,
+  "Groups": ["example-seal-group-1"],
+  "AllowTo": ["example-seal-group-1"]
+}
 ```
-Storage miner
-Path with Groups: ["example-storage-group-1"]
-Worker 1 (PC1, PC2):
-Path with Groups: ["example-seal-group-1"], AllowTo: ["example-seal-group-1"]
-Worker 2 (PC1, PC2):
-Path with Groups: ["example-seal-group-2"], AllowTo: ["example-seal-group-2"]
-Worker 3 (PC1):
-Path with AllowTo: ["example-seal-group-1""]
+- Worker 2 (PC1, PC2):
+```json
+{        
+  "ID": "b5db38b9-2d2e-06eb-8367-7338e1bcd0f1",
+ "Weight": 10,
+  "CanSeal": true,
+  "CanStore": false,
+  "MaxStorage": 0,
+  "Groups": ["example-seal-group-2"],
+  "AllowTo": ["example-seal-group-2"]
+}
 ```
 
-Without storage groups, PC2 tasks on workers 1 or 2 could be scheduled with sector data from other workers, which would often waste bandwidth and occasionally block processing on fetching data.
+- Worker 3 (PC1):
+```json
+{        
+  "ID": "423e45b7-e8e6-64b5-9a62-eb45929d9562", 
+  "Weight": 10,
+  "CanSeal": true,
+  "CanStore": false,
+  "MaxStorage": 0,
+  "Groups": null,
+  "AllowTo": ["example-seal-group-1"]
+}
+```
+Worker 1 and Worker 2 are given unique `Groups` values, and each worker's `AllowTo` matches its respective `Groups`. This ensures PC1 and PC2 for any given sector always execute on the same worker. 
 
-With storage groups configured as above, sectors which had PC1 done on worker1 / worker2 will always execute PC2 on the same worker. Sectors from worker3 will only go to worker1 for PC2
+Setting `AllowTo` on Worker 3 to Worker 1's `Groups` value means sectors from Worker 3 will only go to Worker 1 for PC2.
 
-Groups can be setup in two ways:
+#### Implementation
 
-- For new storage paths, set with
-``` 
-lotus-miner storage attach --init --groups <["group-a", "group-b" ... ]>, --allow-to=<["group-a", "group-b" ... ]> or lotus-worker storage attach --init --groups <["group-a", "group-b" ... ]>, --allow-to <["group-a", "group-b" ... ]>
-```
-- For existing storage paths, modify [path]/sectorstore.json, then restarting lotus-miner/worker
+To set up storage groups when initializing new storage paths: 
+```shell
+lotus-miner storage attach --init --seal [--groups <group-a>] [--groups <group-b>] [--allow-to <group-a>] [--allow-to <group-b>] /path/to/storage
+# or 
+lotus-worker storage attach --init --seal [--groups <group-a>] [--groups <group-b>] [--allow-to <group-a>] [--allow-to <group-b>] /path/to/storage
 ```
+For existing storage paths, add the lines to `sectorstore.json` in each sealing storage location, then restart `lotus-worker`. (The storage locations used by a worker can be found in `LOTUS_WORKER_PATH/storage.json`.)
+```json
```

Comment:
````suggestion
lotus-worker storage attach --init --seal [--groups <group-a>] [--groups <group-b>] [--allow-to <group-a>] [--allow-to <group-b>] /path/to/storage
```

For existing storage paths, add the lines to `sectorstore.json` in each sealing storage location, then restart `lotus-worker`. (The storage locations used by a worker can be found in `LOTUS_WORKER_PATH/storage.json`.)

```json
````